src/app/legacy/containers/PageHandlers/withData/index.jsx

```diff
@@ -1,16 +1,25 @@
 import React, { useContext } from 'react';
-import { element, string, number } from 'prop-types';
+import { element, string, number, shape } from 'prop-types';
 import ErrorPage from '#pages/ErrorPage/ErrorPage';
 import { pageDataPropType } from '#models/propTypes/data';
 import { ServiceContext } from '../../../../contexts/ServiceContext';
+import { RequestContext } from '../../../../contexts/RequestContext';
 import shouldRender from './shouldRender';
 
 const WithData = Component => {
   const DataContainer = ({ pageData, status, ...props }) => {
     const { service } = props;
+    const { pathname } = props.location;
```

Comment:
can we do `const { service,  location.pathname: pathname} = props;`? ( no idea if this works, btw!)

Reply:
Good thinking - I have refactored